docs/what-is-nuget.md

```diff
@@ -32,7 +32,7 @@ A "compatible" package means that it contains assemblies built for at least one
 Package developers who require APIs outside of .NET Standard, on the other hand, create separate assemblies for the different target frameworks they want to support and include all of those assemblies in the same package (which is called "multi-targeting"). When a consumer installs such a package, NuGet extracts only those assemblies that are needed by the project. This minimizes the package's footprint in the final application and/or assemblies produced by that project. A multi-targeting package is, of course, more difficult for its creator to maintain.
 
 > [!Note]
-> Targeting .NET Standard supersedes the previous approach of using various portable class library (PCL) targets. This documentation therefore focuses on creating packages for .NET Standard.
+> For guidance on app components vs. reusable libraries, see the [.NET Standard documentation on the topic](/dotnet/standard/net-standard?branch=master#when-to-target-net50-vs-netstandard).
```

Comment:
My only thought here is that "app components" may not be a readily understood term and I didn't see it used elsewhere in the doc on a quick read. People may not understand the comparison.

Reply:
You can find that term on the linked page: https://docs.microsoft.com/en-us/dotnet/standard/net-standard?branch=master#when-to-target-net50-vs-netstandard